charts/enterprise-metrics/capped-large.yaml

```diff
@@ -0,0 +1,144 @@
+# These values configure the Grafana Enterprise Metrics cluster to
+# handle production ingestion of ~10M active series using the blocks
+# storage engine.
+# Query requirements can vary dramatically depending on query rate and query
+# ranges. The values here satisfy a "usual" query load as seen from our
+# production clusters at this scale.
+# It is important to ensure that you run no more than one ingester replica
+# per node so that a single node failure does not cause data loss. Zone aware
+# replication can be configured to ensure data replication spans availability
+# zones. Refer to [Zone Aware Replication](https://cortexmetrics.io/docs/guides/zone-aware-replication/)
+# for more information.
+# Minio is no longer enabled and you are encouraged to use your cloud providers
+# object storage service for production deployments.
+admin_api:
+  replicas: 3
+  resources:
+    requests:
+      cpu: 1
```

Comment:
I noticed that the requests have been bumped from the `200m` in `large.yaml` to `1` in this configuration, was this intended and do we want to update `large.yaml` too?

Reply:
GEM is billed by CPU cores, which means you'd pay for a full core, even if you'd limit the container to only a fraction of it. Therefore the maximization.